k8s/production/runners/protected/pcluster/graviton/3/release.yaml

```diff
@@ -0,0 +1,139 @@
+
+            namespace = "pipeline"
+            poll_timeout = 600  # ten minutes
+            service_account = "runner"
+
+            # Place pod close to other pipeline pods if possible ("pack" the pods tightly)
+            # Docs: https://docs.gitlab.com/runner/executors/kubernetes.html#define-nodes-where-pods-are-scheduled
+            [runners.kubernetes.affinity]
+              [runners.kubernetes.affinity.pod_affinity]
+                [[runners.kubernetes.affinity.pod_affinity.preferred_during_scheduling_ignored_during_execution]]
+                weight = 1
+                [runners.kubernetes.affinity.pod_affinity.preferred_during_scheduling_ignored_during_execution.pod_affinity_term]
+                  topology_key = "topology.kubernetes.io/zone"
+                  [runners.kubernetes.affinity.pod_affinity.preferred_during_scheduling_ignored_during_execution.pod_affinity_term.label_selector]
+                    [[runners.kubernetes.affinity.pod_affinity.preferred_during_scheduling_ignored_during_execution.pod_affinity_term.label_selector.match_expressions]]
+                      key = "spack.io/runner"
+                      operator = "In"
+                      values = ["true"]
+
+            [runners.kubernetes.pod_annotations]
+              "fluentbit.io/exclude" = "true"
+              "karpenter.sh/do-not-evict" = "true"
+              "gitlab/ci_pipeline_url" = "$CI_PIPELINE_URL"
+              "gitlab/ci_job_url" = "$CI_JOB_URL"
+              "gitlab/ci_project_url" = "$CI_PROJECT_URL"
+              "gitlab/ci_runner_description" = "$CI_RUNNER_DESCRIPTION"
+              "gitlab/ci_job_id" = "$CI_JOB_ID"
+            [runners.kubernetes.pod_labels]
+              "spack.io/runner" = "true"
+              "gitlab/ci_job_size" = "$CI_JOB_SIZE"
+              "metrics/gitlab_ci_pipeline_id" = "$CI_PIPELINE_ID"
+              "metrics/gitlab_ci_project_namespace" = "$CI_PROJECT_NAMESPACE"
+              "metrics/gitlab_ci_project_name" = "$CI_PROJECT_NAME"
+              "metrics/gitlab_ci_job_stage" = "$CI_JOB_STAGE"
+              "metrics/gitlab_ci_commit_ref_name" = "$CI_COMMIT_REF_NAME"
+              "metrics/spack_ci_stack_name" = "$SPACK_CI_STACK_NAME"
+              "metrics/spack_job_spec_pkg_name" = "$SPACK_JOB_SPEC_PKG_NAME"
+              "metrics/spack_spec_needs_rebuild" = "$SPACK_SPEC_NEEDS_REBUILD"
+            [runners.kubernetes.node_selector]
+              "spack.io/node-pool" = "pcluster-amzn2-glr-graviton3"
+
+            [[runners.kubernetes.volumes.secret]]
+              name = "spack-intermediate-ci-signing-key"
+              mount_path = "/mnt/key/"
+              read_only = true
+            [[runners.kubernetes.volumes.host_path]]
+              name = "software"
+              mount_path = "/opt"
+              read_only = true
+            [[runners.kubernetes.volumes.host_path]]
+              name = "keys"
+              mount_path = "/mnt/key"
```

Comment:
I wonder if this is overwriting the `spack-intermediate-ci-signing-key` stuff mounted in the same location above?  See this job, where there don't seem to be any keys there:

https://gitlab.spack.io/spack/spack/-/jobs/7122837#L404

Reply:
Delete this, use the mounted secret > the host path. Host path is the AMI location, which doesn't have secrets.